### ISSUE TITLE
Abnormal circulating Placental Growth Factor concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6004]
**Preferred term label:** Abnormal circulating Placental Growth Factor concentration

**Synonyms** Abnormal PlGF; Abnormal Placenta Growth Factor

**Definition (free text, please give PubMed ID)** Concentration of placenta growth factor in the blood is outside the normal range., PMID: 29115294

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876; Elevated vascular endothelial growth factor level HP:0031052

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
No normal ranges have been determined for this protein in serum. For instance, there is no LOINC term for this test. Therefore, it is possible to do a cohort study that shows a higher level in individuals with some condition, but it is not possible to do a test in an individual and say that the level is "too high" or "too low". This means that this would not be a valid HPO term.
